packages/enzyme-test-suite/test/Utils-spec.jsx

```diff
@@ -600,7 +600,11 @@ describe('Utils', () => {
         Foo.displayName = 'CustomWrapper';
 
         const MemoForwardFoo = React.memo(React.forwardRef(Foo));
-        expect(adapter.displayNameOfNode(<MemoForwardFoo />)).to.equal('Memo(ForwardRef(CustomWrapper))');
+        if (is('>= 17')) {
+          expect(adapter.displayNameOfNode(<MemoForwardFoo />)).to.equal('Memo([object Object])');
```

Comment:
The object looks like this:
<img width="424" alt="Screen Shot 2021-08-11 at 4 03 27 PM" src="https://user-images.githubusercontent.com/1695880/129377009-bc37bb5d-7e39-40bb-adbb-329b60246d05.png">